config.go

```diff
@@ -123,6 +124,10 @@ type Config struct {
 	// a default logger will be created.
 	Logger *zap.Logger
 
+	// Set a http proxy to use when issuing a certificate.
+	// Default is http.ProxyFromEnvironment
+	HttpProxy func(*http.Request) (*url.URL, error)
```

Comment:
```suggestion
	HTTPProxy func(*http.Request) (*url.URL, error)
```

In this house, we capitalize initialisms :upside_down_face: 

Also, I think this would be better placed in the ACMEIssuer struct, as it's used only for ACME transactions.

Reply:
Ok, I will change both.
When in ACMEIssuer, I can either default to `http.ProxyFromEnvironment` (to be backwards-compatible), but this way there is no option to disable the proxy.
The other option would be to default to `nil`, which allows to explicitly disable the proxy for ACME requests, but slightly changes backwards compatibility.